shared/cfg/guest-hw.cfg

```diff
@@ -163,7 +163,6 @@ variants:
         # available, it will also try to get first available pf.
         device_name = eth1
         nettype = bridge
-        pf_filter_re = "82576 Gigabit Network"
```

Comment:
other sriov tests also uses this param, we can override it in the --vt-config, what do you say ?